packages/testing/playwright/workflows/Test_workflow_2.json

```diff
@@ -0,0 +1,58 @@
+{
+	"name": "Test workflow 2",
+	"nodes": [
+		{
+			"parameters": {},
+			"id": "624e0991-5dac-468b-b872-a9d35cb2c7d1",
+			"name": "On clicking 'execute'",
+			"type": "n8n-nodes-base.manualTrigger",
+			"typeVersion": 1,
+			"position": [360, 260]
+		},
+		{
+			"parameters": {
+				"jsCode": "// Loop over input items and add a new field\n// called 'myNewField' to the JSON of each one\nfor (const item of $input.all()) {\n  item.json.myNewField = 1;\n}\n\nreturn $input.all();"
+			},
+			"id": "48823b3a-ec82-4a05-84b8-24ac2747e648",
+			"name": "Code",
+			"type": "n8n-nodes-base.code",
+			"typeVersion": 1,
+			"position": [580, 260]
+		}
+	],
+	"pinData": {},
+	"connections": {
+		"On clicking 'execute'": {
+			"main": [
+				[
+					{
+						"node": "Code",
+						"type": "main",
+						"index": 0
+					}
+				]
+			]
+		}
+	},
+	"active": false,
+	"settings": {},
+	"hash": "4d2e29ffcae2a12bdd28a7abe9681a6b",
+	"id": 4,
+	"meta": {
+		"instanceId": "a5280676597d00ecd0ea712da7f9cf2ce90174a791a309112731f6e44d162f35"
```

Comment:
Not sure we need `instanceId` in tests (also maybe `hash`). This is really a nitpick but I think we should be making these fixtures as small as possible and also discourage adding any potential PII inside unless really necessary.